src/or/router.c

```diff
@@ -114,6 +114,56 @@ const char *format_node_description(char *buf,
                                     const tor_addr_t *addr,
                                     uint32_t addr32h);
 
+/** Return a readonly string with human readable description
+ * of <b>err</b>.
+ */
+const char *
+routerinfo_errno_to_string(int err)
```

Comment:
Hmm, `errno` is usually quite specific to the libc error values.

I would recommend just using `err`.

Reply:
Fixed in 8ad97b7dcf143fc488422f9866613fea3d900119.